django/db/migrations/state.py

```diff
@@ -226,6 +225,17 @@ def __init__(self, real_apps, models, ignore_swappable=False):
             labels = (".".join(model_key) for model_key in self._pending_operations)
             raise ValueError(msg % ", ".join(labels))
 
+    def bulk_update(self):
```

Comment:
I think you can simply decorate the method with `@contextmanager`.
Also, a docstring for the method would be good to add.